demo/DemoArticles/en.Audioslave.json

```diff
@@ -0,0 +1 @@
me becomes the music and the people.</p>\n</blockquote>\n<p>After the name was announced, it emerged that it was already being used by an unsigned band from <a href=\"/wiki/Liverpool\" title=\"Liverpool\">Liverpool</a>. The two bands worked out a settlement, with the American Audioslave paying $30,000 in a deal that allowed each band to use the name. To avoid confusion, the Liverpool band would rename themselves \"The Most Terrifying Thing.\"<sup id=\"cite_ref-Yahoo.21_Music_-_Audioslave_Works_Out_Settlement_To_Share_Name_With_U.K._Band_98-1\" class=\"reference\"><a href=\"#cite_note-Yahoo.21_Music_-_Audioslave_Works_Out_Settlement_To_Share_Name_With_U.K._Band-98\">[98]</a></sup></p>\n<p>The name was mocked by critics due to its uninspired nature,<sup id=\"cite_ref-99\" class=\"reference\"><a href=\"#cite_note-99\">[99]</a></sup> and was regarded as one of the worst in contemporary rock music,<sup id=\"cite_ref-100\" class=\"reference\"><a href=\"#cite_note-100\">[100]</a></sup> or even of all time.<sup id=\"cite_ref-101\" class=\"reference\"><a href=\"#cite_note-101\">[101]</a></sup> <i><a href=\"/wiki/Pitchfork_(website)\" title=\"Pitchfork (website)\">Pitchfork</a></i> called it the \"most asinine bandname of the year,\"<sup id=\"cite_ref-pitchfork_37-1\" class=\"reference\"><a href=\"#cite_note-pitchfork-37\">[37]</a></sup> and <a href=\"/wiki/Chuck_Klosterman\" title=\"Chuck Klosterman\">Chuck Klosterman</a> of <i><a href=\"/wiki/Spin_(magazine)\" title=\"Spin (magazine)\">Spin</a></i> magazine chided it as \"one of the dumbest band names in recent rock history.\"<sup id=\"cite_ref-102\" class=\"reference\"><a href=\"#cite_note-102\">[102]</a></sup></p>"},{"toclevel":1,"level":"2","line":"Musical style and influences","number":"3","index":"9","fromtitle":"Audioslave","anchor":"Musical_style_and_influences","id":9,"text":"<p>By combining '70s style hard rock riffing with alternative rock, Audioslave created a distinctive sound.<sup id=\"cite_ref-Plugged_In_Online_-_Audioslave_Comes_.27Out_of_Exile.27_103-0\" class=\"reference\"><a href=\"#cite_note-Plugged_In_Online_-_Audioslave_Comes_.27Out_of_Exile.27-103\">[103]</a></sup> This mix was driven by Cornell's wide vocal range,<sup id=\"cite_ref-104\" class=\"reference\"><a href=\"#cite_note-104\">[104]</a></sup> Morello's innovative guitar solos and the robust rhythm section of Wilk and Commerford.<sup id=\"cite_ref-105\" class=\"reference\"><a href=\"#cite_note-105\">[105]</a></sup> Morello, although stating he \"never felt musically limited\" in Rage Against the Machine, did say that he had \"a lot more scope to explore with Audioslave\" and a \"wider musical territory.\"<sup id=\"cite_ref-106\" class=\"reference\"><a href=\"#cite_note-106\">[106]</a></sup> This meant that the instrumentalists had the opportunity to write slow and melodic songs, something they had not done before.</p>\n<p>As opposed to de la Rocha's lyrics, Cornell's were mostly apolitical; Morello referred to them as \"haunt
```

Comment:
Since this is nonproduction code, please format these JSON samples for easy reading. I use [a bookmarklet](https://gist.github.com/niedzielski/53c98af986955053aaabcc8a19a62f39) when viewing responses in browser but there are many other ways to go about it.